contrib/JavaGraphqlClient/src/main/graphql/schema.json

```diff
@@ -0,0 +1 @@
ll},{"name":"domain","description":"Urn of the domain to fetch analytics for (If empty or GLOBAL, queries across all domains)","type":{"kind":"SCALAR","name":"String","ofType":null},"defaultValue":null},{"name":"query","description":"Search query to filter down result (If empty, does not apply any search query)","type":{"kind":"SCALAR","name":"String","ofType":null},"defaultValue":null}],"interfaces":null,"enumValues":null,"possibleTypes":null},{"kind":"OBJECT","name":"Metrics","description":null,"fields":[{"name":"performanceMeasures","description":"Measures of ML Model performance","args":[],"type":{"kind":"LIST","name":null,"ofType":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}}},"isDeprecated":false,"deprecationReason":null},{"name":"decisionThreshold","description":"Decision Thresholds used if any","args":[],"type":{"kind":"LIST","name":null,"ofType":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}}},"isDeprecated":false,"deprecationReason":null}],"inputFields":null,"interfaces":[],"enumValues":null,"possibleTypes":null},{"kind":"OBJECT","name":"Mutation","description":"Root type used for updating DataHub Metadata\nComing soon createEntity, addOwner, removeOwner mutations","fields":[{"name":"updateDataset","description":"Update the metadata about a particular Dataset","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}},"defaultValue":null},{"name":"input","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"INPUT_OBJECT","name":"DatasetUpdateInput","ofType":null}},"defaultValue":null}],"type":{"kind":"OBJECT","name":"Dataset","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"updateChart","description":"Update the metadata about a particular Chart","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}},"defaultValue":null},{"name":"input","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"INPUT_OBJECT","name":"ChartUpdateInput","ofType":null}},"defaultValue":null}],"type":{"kind":"OBJECT","name":"Chart","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"updateDashboard","description":"Update the metadata about a particular Dashboard","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"SCALAR","name":"String","ofType":null}},"defaultValue":null},{"name":"input","description":null,"type":{"kind":"NON_NULL","name":null,"ofType":{"kind":"INPUT_OBJECT","name":"DashboardUpdateInput","ofType":null}},"defaultValue":null}],"type":{"kind":"OBJECT","name":"Dashboard","ofType":null},"isDeprecated":false,"deprecationReason":null},{"name":"updateNotebook","description":"Update the metadata about a particular Notebook","args":[{"name":"urn","description":null,"type":{"kind":"NON_NULL"
```

Comment:
does it make sense to keep this under version control? won't it immediately become out of date if we update the graphql spec?

Reply:
I thought of keeping it for reference.I remove it